src/ray/stats/metric.cc

```diff
@@ -24,8 +24,10 @@ namespace stats {
 
 absl::Mutex Metric::registration_mutex_;
 
-static void RegisterAsView(opencensus::stats::ViewDescriptor view_descriptor,
-                           const std::vector<opencensus::tags::TagKey> &keys) {
+namespace details {
```

Comment:
Btw, can you use `internal` instead of details? We talked about it with @scv119 and decided to use internal for this (since it seems like already used in some places).